spec/controllers/idv/review_controller_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+describe Idv::ReviewController do
+  render_views
```

Comment:
As mentioned before, calling `render_views` slows down tests, and it's only needed if you want to see text in the view, so it's rarely a good idea to call it at the top of a controller spec, because it unnecessarily slows down specs that don't test the view.